src/js/view/controls/rightclick.js

```diff
@@ -127,8 +137,11 @@ export default class RightClick {
         this.mouseOverContext = false;
         this.layer = layer;
 
-        // Defer the rest of setup until the first click
-        _playerElement.oncontextmenu = this.rightClick.bind(this);
+        // iOS does not support oncontextmenu
+        if (!OS.iOS) {
```

Comment:
There is no harm in setting this on iOS, even though it is not supported. It costs more to import environment and check for iOS than it does to just set `oncontextmenu`.